graph/src/blockchain/empty_runtime_adapter.rs

```diff
@@ -0,0 +1,22 @@
+use std::marker::PhantomData;
+
+use super::{Blockchain, HostFn, RuntimeAdapter};
+
+/// A [`RuntimeAdapter`] that does not expose any host functions.
+#[derive(Debug, Clone)]
+pub struct EmptyRuntimeAdapter<C>(PhantomData<C>);
```

Comment:
What do you think about `NoopRuntimeAdapter`?

Reply:
Done!